_maps/asteroidstation.dm

```diff
@@ -0,0 +1,31 @@
+/*
+The /tg/ codebase currently requires you to have 7 z-levels of the same size dimensions.
+z-level order is important, the order you put them in inside this file will determine what z level number they are assigned ingame.
+Names of z-level do not matter, but order does greatly, for instances such as checking alive status of revheads on z1
+*/
+
+#if !defined(MAP_FILE)
+
+		#define TITLESCREEN "title" //Add an image in misc/fullscreen.dmi, and set this define to the icon_state, to set a custom titlescreen for your map
+
+		#define MINETYPE "mining"
+
+        #include "map_files\AsteroidStation\AsteroidStation.dmm"
+        #include "map_files\generic\z2.dmm"
+        #include "map_files\generic\z3.dmm"
+        #include "map_files\generic\z4.dmm"
+        #include "map_files\AsteroidStation\z5.dmm"
+        #include "map_files\generic\z6.dmm"
+        #include "map_files\generic\z7.dmm"
```

Comment:
Jesus, when did these happen, also the map works fine without them. But I will add tomorrow, for now DNM I guess.